src/composeapp.cc

```diff
@@ -42,6 +44,68 @@ void ComposeApp::remove() {
   }
 }
 
+bool ComposeApp::check() {
+  bool result = false;
+
+  do {
+    bool cmd_res{false};
+    std::string cmd_output;
+
+    std::tie(cmd_res, cmd_output) = cmd("config");
+    if (!cmd_res) {
+      LOG_WARNING << "Missing and incomplete App config: " << name_;
+      break;
+    }
+
+    // calculate a number of container images that a given App consists of
+    std::string::size_type find_pos{0};
+    int expected_image_number{0};
+
+    while ((find_pos = cmd_output.find("image:", find_pos)) != std::string::npos) {
```

Comment:
I suppose we don't have a YAML parser available. Should be fine. But - you should check to make sure this line doesn't start with a `#`. I could see people doing something like:
~~~
services:
  andy:
    # image: foo-devel
    image: foo
~~~

Reply:
ok

Reply:
There are C/C++ yaml parsers, of course, I just didn't want to introduce another dependency.

Reply:
no - thanks for not!

Reply:
> But - you should check to make sure this line doesn't start with a `#`. I could see people doing something like:

done